robboat/views.py

```diff
@@ -62,7 +61,11 @@ def webhook(request):
         return HttpResponseBadRequest('Malformed filespec')
     sha, filepath, start, end = filespec.groups()
     start, end = map(int, (start, end))
-    instruction = '\n'.join(all_lines[2:])
+    # Ignore if we're not mentioned
```

Comment:
```suggestion    
    instruction = '\n'.join(all_lines[2:])
```

Let's stick with defining down here for now. Also, appreciate the comment, but don't need it since the rest isn't commented.